src/objects/Attribute__c.object

```diff
@@ -99,11 +99,46 @@
                     <default>false</default>
                     <label>Accommodation</label>
                 </value>
+                <value>
```

Comment:
@lifewithryan  I don't know if it matters if these are alphabetical or not, but the first 4 pre-existing options are not alphabetical while the rest of these are.

Reply:
I noticed that and didn’t touch it because that’s how it came down. Shouldn’t hurt but for craftsmen purposes I’ll go on and fix tonight or Friday morning.

Reply:
deploying and re-pulling still provided this as the default order so I'm not goin to touch it. I don't believe it really matters anyway (apparently). The deploy/pull process (in my experience) has normally cleaned things up. It didn't change anything here so, I'd say we're good.